src/components/CommentCard.jsx

```diff
@@ -15,6 +39,19 @@ export const CommentCard = ({
         Votes: <span className="comment-votes">{votes}</span>
       </p>
       <p className="time-date">Published on: {dateAndTime.date}</p>
+      {user === author ? (
```

Comment:
A nice approach to making sure only the author of the comment is able to delete it 👍